src/bundle/Resources/views/content/modal_draft_conflict.html.twig

```diff
@@ -6,7 +6,7 @@
     <div class="modal-dialog modal-lg" role="document">
         <div class="modal-content">
             <div class="modal-header">
-                <h5>{{ 'draft.conflict.header'|trans|desc('Draft conflict when editing') }}</h5>
+                <h5 class="modal-title">{{ 'draft.conflict.header'|trans|desc('Draft conflict when editing') }}</h5>
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                     <span aria-hidden="true">&times;</span>
```

Comment:
Shouldn't we use SVG icon instead of HTML entitty for the close button symbol, the `x` sign?

Reply:
@sunpietro 👍 that's going to be covered in a different PR (Update closing buttons for modals)